packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-policy-drilldown.vue

```diff
@@ -404,5 +406,16 @@ onMounted(() => {
 <style scoped>
 ul {
 	list-style: none;
+
+	& + span {
+		margin-top: var(--gap-2);
+	}
```

Comment:
Maybe instead of using a `<span>` you can just add a classname `mt-2` to the button iteself.